app/src/main/java/org/sinisterstuf/guesstheanimal/ui/GreetingActivity.java

```diff
@@ -1,11 +1,15 @@
-package org.sinisterstuf.guesstheanimal;
+package org.sinisterstuf.guesstheanimal.ui;
 
 import android.app.Activity;
 import android.content.Intent;
 import android.os.Bundle;
 import android.view.View;
 
-public class Greeting extends Activity {
+import org.sinisterstuf.guesstheanimal.Animal;
+import org.sinisterstuf.guesstheanimal.Game;
+import org.sinisterstuf.guesstheanimal.R;
```

Comment:
These imports got added but I can't see where they're used...

Reply:
Now I see. They weren't needed before because they were in the same package.

Reply:
They are used, but this file moved to another package, so it can't refer to those classes